CMakeLists.txt

```diff
@@ -7,17 +7,22 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR})
 enable_language(CXX)
 
 find_package(NFS REQUIRED)
-find_package(kodi REQUIRED)
+find_package(Kodi REQUIRED)
+find_package(p8-platform REQUIRED)
 find_package(kodiplatform REQUIRED)
 
-include_directories(${KODI_INCLUDE_DIR}
-                    ${KODIPLATFORM_INCLUDE_DIRS}
+include_directories(${KODI_INCLUDE_DIR}/.. # Hack way with "/..", need bigger Kodi cmake rework to match right include ways
```

Comment:
??

Reply:
generic meh with new add-on bindings. ask @AlwinEsch when it will be resolved and how to resolve it..

Reply:
Yeahh, the stupid bad thing. The old addon system has always included the kodi folder itself. 

For a `#include "libKODI_vfs.h"` was it OK. But with the new where it then is `#include "Filesystem.h"` only can it be complicated with other headers from somewhere else.

Thats why new ones should always use a `#include <kodi/Filesystem.h>`.

But as long there still addon systems (PVR and Game) is this bad thing needed to have right folder for here and on the old ones.

After them are replaced becomes the cmake part changed.